addons/mail/static/tests/legacy/activity/activity_mark_done_popover_tests.js

```diff
@@ -17,6 +17,7 @@ QUnit.test("activity mark done popover simplest layout", async () => {
         can_write: true,
         res_id: partnerId,
         res_model: "res.partner",
+        user_id: pyEnv.currentUserId,
```

Comment:
This should be a default in model definition rather than added manually to every test (and it matches python).